browser/src/canvas/sections/AutoFillMarkerSection.ts

```diff
@@ -58,14 +57,7 @@ class AutoFillMarkerSection extends CanvasSectionObject {
 			this.size = [Math.round(16 * app.dpiScale), Math.round(16 * app.dpiScale)];
 		}
 
-		app.events.on('updatepermission', this.setMaybeReadonlyShowSection.bind(this));
-
-		// HACK: used when dealing with CanvasSectionContainer only having setShowSection via addSectionFunctions
-		// we need to rename the property otherwise
-		// (1) we will not be able to call it again via super, since as it's not a real superclass function
-		// (2) we will not be able to call our own setShowSection as, since as it's set directly on the object, it'll override everything in the prototype...
-		this.super_setShowSection = this.setShowSection;
-		delete this.setShowSection;
+		app.events.on('updatepermission', this.setShowSection.bind(this));
```

Comment:
This doesn't do the same thing anymore - it'll call `this.setShowSection` with the update permission event parameter, setting `this._showSection` to that. As objects are always truthy, this is roughly equivalent to calling `this.setShowSection(true)`. If the section was *not* shown before, this will clobber that when you enter and then leave readonly mode.

Reply:
Thanks for noticing. Fixed the behaviour.